rllib/models/catalog.py

```diff
@@ -345,7 +345,7 @@ def get_action_dist(
                 "Unsupported args: {} {}".format(action_space, dist_type)
             )
 
-        return dist_cls, dist_cls.required_model_output_shape(action_space, config)
+        return dist_cls, int(dist_cls.required_model_output_shape(action_space, config))
```

Comment:
This seems unrelated but is due to the fact that we calculate the output shape from spaces that may contain non Python int values, but rather np.int64 values ->

This commit in gymnasium: https://github.com/Farama-Foundation/Gymnasium/pull/141/files#diff-08f7c040be788bbfb4790fc76ac8837b74532e4977e4ef5d444d7b008a029e00R45
causes this issue for us.
So I'm proposing this change to force that we return the type promised in this method's signature.